.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Ask a Question
+    url: https://lightning.ai/forums
```

Comment:
do we use forums? we created [reddit](https://www.reddit.com/r/lightningAI/) recently.

Reply:
this may be updated everywhere